### ISSUE TITLE
Netboot.xyz hangs on EFI boot in Virtualbox 6.1.26

### ISSUE THREAD

**Issue by tayrayb** [netbootxyz/netboot.xyz#1003]
As the title says, Netboot hangs when booting the BOOTX64.EFI from the EFI shell in Virtualbox.

**To Reproduce**
1. Navigate to BOOTX64.EFI in EFI\BOOT in the EFI shell.

**Expected behavior**
Locks up on iPXE initializing devices.

**Screenshots**
![image](https://user-images.githubusercontent.com/10287602/134452482-1a79eb5b-42ab-46a9-83ae-2cbbbfc12fbe.png)

**Additional context**
My CPU is an AMD Ryzen 9 3900X, not sure if that information helps. I also have made sure SVM and all the needed stuff is enabled in the BIOS as well.

**Comment by antonym** [comment]
Have you tried the netboot.xyz ISO directly in VirtualBox to see if that path is successful? There typically should be no interaction with UEFI required until you hit the netboot.xyz menu.

**Comment by tayrayb** [comment]
No, it just boots straight into the UEFI shell like this.

![image](https://user-images.githubusercontent.com/10287602/135400645-e929f830-d162-4f32-9a03-5e28f56d9d0a.png)

I've done this before, I'm sure I remember I did the same with older versions of Virtualbox, having to navigate to the BOOTX64.EFI manually.

It just hangs when its booted manually on the latest Virtualbox with the CPU maxed out.

**Comment by antonym** [comment]
You may need to adjust your boot order, if you are getting the UEFI shell first, it's more than likely skipping the ISO or network boot options.

In my old setup on Virtual Box, I'd add the ISO, set CDROM to boot first and then it would immediately boot into netboot.xyz. You shouldn't see the UEFI shell and shouldn't need to interact with anything until the menu loads.

**Comment by tayrayb** [comment]
That's the bizarre part, I tried setting the Optical drive only to boot and it still drops straight into the EFI shell. Booting Non-EFI works perfectly fine and will boot straight up into netboot.xyz. I also tried booting from the bios menu in Virtualbox as well.
![image](https://user-images.githubusercontent.com/10287602/138787770-2c7990ae-269f-45e5-a6ac-c9150906b455.png)
But it just does nothing, the screen flickers and then just goes back to the boot menu. Not sure whats going on, but I'm suspecting its just the Virtualbox EFI shell just not handling it correctly, but I think I'll just close the issue until I can figure out a workaround.
